tests/system/alerts/test_alerts.py

```diff
@@ -84,6 +85,58 @@ def test_job_failure_alert(self):
         self._validate_notifications_on_nuclio(
             nuclio_function_url, expected_notifications
         )
+        # get all activations
```

Comment:
i would move all of this to a separate function ( _validate_alert_activations ? )
so the test will not be so long and if we want to add more checks to it in the future it would be more readable

Reply:
@Yacouby what exactly do you want to move here? Every check here is specific to this test